lib/pbench/client/__init__.py

```diff
@@ -0,0 +1,289 @@
+from typing import Dict, List, Optional, Union
+from urllib import parse
+
+import requests
+
+
+# A type defined to conform to the semantic definition of a JSON structure
+# with Python syntax.
+JSONSTRING = str
+JSONNUMBER = Union[int, float]
+JSONVALUE = Union["JSONOBJECT", "JSONARRAY", JSONSTRING, JSONNUMBER, bool, None]
+JSONARRAY = List[JSONVALUE]
+JSONOBJECT = Dict[JSONSTRING, JSONVALUE]
+JSON = JSONVALUE
```

Comment:
I don't really understand Python packaging...you're saying that, because this file is named `lib/pbench/client/__init__.py` it is completely independent of definitions in `lib/pbench/__init__.py`?

Reply:
The idea is that we *can* package this separately... and probably should... but I didn't try to do that in this PR, just for convenience. (I.e., for functional tests I don't think it matters, but if we want to expose this as a general Python client we should package it separately. In any case, I'd rather avoid internal code dependencies.)